src/Spryker/Zed/EventBehavior/Dependency/Plugin/EventResourceQueryContainerPluginInterface.php

```diff
@@ -16,7 +16,9 @@
      *
      * @api
      *
+     * @param int[] $ids
+     *
      * @return \Propel\Runtime\ActiveQuery\ModelCriteria|null
      */
-    public function queryData();
+    public function queryData($ids = []);
```

Comment:
Why have a default value in here? .. also you can follow the PHP7 convention `?array $ids = []` and the docblock like so `@param int[]|null`.

Reply:
The default value is only allowed if any caller could omit() the argument and the function call still makes sense.

Reply:
@m7moud @dereuromark , default value in here because it`s supposed to call with and without parameter: 
https://github.com/spryker/event-behavior/blob/feature/ps-1995-event-resource-plugins/src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerManager.php#L92

and here:

https://github.com/spryker/event-behavior/blob/feature/ps-1995-event-resource-plugins/src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerManager.php#L76

Reply:
👍 that's ok I guess, please change the docsblock and typehint.